pkg/status/status.go

```diff
@@ -138,3 +143,39 @@ func GetFromStatuses(statuses map[string]*string) (status *common.Status) {
 
 	return
 }
+
+// HealthState requires custom parsing as it is a proto enum. Otherwise enum 0 (NOT_KNOWN) will be an empty string.
+func parseHealth(hstate *common.Health) string {
+	if hstate == nil {
+		return fmt.Sprintf("currentState:%s", common.HealthState_name[0])
```

Comment:
Can we use the const `HealthState_NOTKNOWN` instead of 0.

Reply:
oh good point yes done